.github/workflows/build.yml

```diff
@@ -126,6 +126,8 @@ jobs:
     runs-on: ubuntu-22.04
     env:
       CHROME_VERSION: ${{ matrix.chrome-version }}
+      JOB_NAME: ${{ matrix.cmd }}-${{ matrix.suite || '' }}${{ matrix.chrome-version == '90' && '-minimum-browser' || '' }}
```

Comment:
There _theoretically_ is a way to get the job name programatically, but that apparently does not work when using strategy matrix. 
[source](https://github.com/actions/toolkit/issues/74#issuecomment-1974028870)